cmd/traffic/cmd/manager/service.go

```diff
@@ -58,6 +58,12 @@ func NewManager(ctx context.Context) (*Manager, context.Context) {
 	}
 	ctx = a8rcloud.WithSystemAPool[managerutil.SystemaCRUDClient](ctx, a8rcloud.TrafficManagerConnName, &ReverseConnProvider{ret})
 	ret.ctx = ctx
+
+	l, err := newLicenseBundleFromDisk("/home/telepresence")
+	if err == nil {
+		ret.license = l
+	}
```

Comment:
pls log the error at an Info level or similar if it's not nil so users can see if/why their license didn't load

Reply:
also this is probably where we should check if we need to get the ClusterID from the license